tests/cli/test_demo.py

```diff
@@ -23,7 +23,7 @@ def test_demo_with_valid_config(mocker, capfd):
         assert main(["demo"]) == 0  # Successful run
 
     out, err = capfd.readouterr()
-    assert out.find("Sending example data to AppSignal...") > 0
+    assert out.find("Sending example data to AppSignal...") > -1
```

Comment:
Isn't `>=` a valid operator in Python?

Reply:
I think you left this comment on an older comment and I already fixed it in a new comment.

Reply:
Do you have a link? That completely disappeared from my head

Reply:
Oh no I got confused with a newer branch that isn't created yet. But I think this is fine. `-1` indicates it couldn't be found. `0` means it's the first character in the string. So if it's larger than -1 it could be found.
`0` worked before because the demo CLI output started with an empty line that I [(re)moved in this change](https://github.com/appsignal/appsignal-python/pull/147/commits/d6089bde712760dc5714b9929d39c2649362d839).

Reply:
My suggestion was to use `>= 0` instead, but it's a personal preference. I'm also good with the `-1`